config/services.yaml

```diff
@@ -216,3 +216,7 @@ services:
     App\Util\FundingOrgFilter:
         bind:
             array $fundingOrgs: "%env(json:FILTER_BY_FUNDING_ORG)%"
+
+    App\Util\FileUploader:
+        bind:
+            $homedirPrefix: "%env(HOMEDIR_PREFIX)%"
```

Comment:
This is ok for now, but the have to make a good set of variables that are named relevant to what they do.